go/test/endtoend/reparent/plannedreparent/reparent_test.go

```diff
@@ -433,7 +435,8 @@ func TestFullStatus(t *testing.T) {
 	utils.ConfirmReplication(t, tablets[0], []*cluster.Vttablet{tablets[1], tablets[2], tablets[3]})
```

Comment:
This change fixes flakiness in `TestFullStatus` where replica status was being checked before it caught up with primary.